### ISSUE TITLE
Simplify deposits UI - EPIC - Overview Balances card (single currency)

### ISSUE THREAD

**Issue by james-allan** [Automattic/woocommerce-payments#5581]
### Description

The goal of this project is to give merchants confidence about where there money is, how they get paid, and when their money will be available.

We will achieve this by implementing an improved design for deposits related information in the payments overview screen (alongside appropriate) documentation.

### Designs

<img width="1168" alt="Screenshot 2023-03-07 at 3 36 54 pm" src="https://user-images.githubusercontent.com/8490476/223330089-13b465c3-84d4-4916-8d02-1b3ab2f5a34b.png">

> **Note**
> The reserve funds section has been cut from scope. 

p2: paJDYF-7fi-p2

### Acceptance criteria

- [x] #5720
- [x] #5721
     - Greeting. "Good afternoon, {user-first-name} 👋". 
     - Woo Payments graphic.
- [x] #5723
    - **Available balance** ⇒ **Available funds**
    - **Pending balance** ⇒ **Pending funds**
    - ~~**Reserve funds**.~~ We don't currently display reserve funds in the UI. The reserve amount is currently not accessible via our `deposits/overview-all` endpoint. The amount reserved will need to added to this endpoint before we can display it in the UI. (see server issue below).
- [x] #5724
- [x] https://github.com/Automattic/woocommerce-payments/issues/5865

#### Removed from scope:

- [x] ~automattic/woocommerce-payments-server/issues/3105~
- [x] ~~#5774~~

> **Note**
> The currency select navigation will be completed as part of a separate epic to support multiple currencies. That epic is still to come, link here once it's created. The work done as part of this epic should take into consideration the placement of that nav.

**Comment by james-allan** [comment]
All tasks have been merged and shipped. Closing this Epic.
